src/singletons/Settings.hpp

```diff
@@ -152,6 +152,8 @@ class Settings
         "/appearance/enableAnimationsWhenFocused", false};
     QStringSetting timestampFormat = {"/appearance/messages/timestampFormat",
                                       "h:mm"};
+    QStringSetting logTimestampFormat = {
+        "/appearance/messages/logTimestampFormat", "hh:mm:ss"};
```

Comment:
Please reformat the setting to take up 4 full lines

```suggestion
    QStringSetting logTimestampFormat = {
        "/appearance/messages/logTimestampFormat",
        "hh:mm:ss",
    };
```

Additionally, the default value here should be the same as we used in logs before (i.e. `HH:mm:ss`)

Reply:
The deleted line 156 shows the value that was used in the logs as `hh:mm:ss`, not `HH:mm:ss`.

Reply:
Ah whoops, I was reading from the suggested change accidently!

Reply:
The important part is that it's not a breaking change, so if it's not breaking this is fine by me